### PR TITLE
Scoped registry

### LINKED ISSUE #8: [Feature Request] Set registry for scoped package
Hi, 

Is it possible to add the scope of package in .npmrc file?

For example, the .npmrc file generated should look like this:

```
@myscope:registry=https://<registry>/
<registry>:_authToken=XXX
```

### PR DESCRIPTION
**Pull request by lantrix**
Adding in scoped registry option. Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,49 +53,65 @@ steps:
           registry: //myprivatenpm.com/
 ```
 
+If you set a registry, you can configure a specific scopes to fetch packages from your custom registries
```
Comment (lantrix) [inline README.md]:
```suggestion
If you set a registry, you can configure a specific scope to fetch packages from your custom registry
```

**Review by 72636c — APPROVED** [review]
Thanks!

**Comment by lantrix** [comment]
We can merge once my tests are completed

**Comment by lantrix** [comment]
We were able to successfully publish with an `.npmrc` set by the pipeline

```yaml
- label: ":books: Publish Package"
  command: npm publish
  plugins:
  - github.com/seek-oss/aws-sm-buildkite-plugin#v2.3.2:
      env:
        NPM_TOKEN: some-read-token
        NPM_PUB_TOKEN: some-publish-token
  - github.com/seek-oss/docker-ecr-cache-buildkite-plugin#v2.2.0:
      secrets: id=npm,src=tmp/.npmrc
      cache-on:
      - package.json
      - pnpm-lock.yaml
  - github.com/seek-oss/private-npm-buildkite-plugin#587a4638:
      env: NPM_PUB_TOKEN
      scope: "@privatescope"
      registry: "//myprivatenpm.com/"
  - github.com/buildkite-plugins/docker-buildkite-plugin#v5.11.0:
      volumes:
      - "/workdir/node_modules"
      environment:
      - NPM_PUB_TOKEN
      - NPM_TOKEN
      mount-buildkite-agent: true
      propagate-environment: true
```

The `.npmrc` created in the project folder was:

```npm
//myprivatenpm.com/:_authToken=<token>
always-auth=true
@privatescope:registry=https://myprivatenpm.com/
```

**Pull request merged**